apps/ledger-live-desktop/src/renderer/hooks/useDeeplinking.js

```diff
@@ -42,18 +42,20 @@ export function useDeepLinkHandler() {
   const walletConnectLiveApp = useFeature("walletConnectLiveApp");
 
   const navigate = useCallback(
-    (url: string, state?: any, search?: string) => {
-      if (url !== location.pathname) {
+    (pathname: string, state?: any, search?: string) => {
+      const hasPathnameChanged = pathname !== location.pathname;
+      const hasSearchChanged = typeof search === "string" && search !== location.search;
```

Comment:
Could it be ```hasNewSearch``` && ```hasNewPathname``` a better name as it is not rewriting the history when "clean" the param?

Reply:
https://github.com/LedgerHQ/ledger-live/pull/2933/commits/5a99ebb5866a3547f8701ee28761feebde728127